templates/test.html

```diff
@@ -208,20 +208,20 @@ <h1>{{ data.player.name }}</h1>
                 </div>
 
                 <div class="gamelist">
-                    {% for game in data.games %}
+                    {% for match in data.matches %}
                     <div class="gamelist-item">
                         <span class="gamelist-item-vs">vs</span>
 
-                        {% for opponent in game.opponents %}
+                        {% for opponent in match.participants %}
```

Comment:
gamelist-item braucht noch match.game.name, aktuell weis man nicht welches spiel dort gespielt wurde in der liste

TODO
- design
- template
- data wire